chainreg/chaincode.go

```diff
@@ -0,0 +1,25 @@
+package chainreg
```

Comment:
nit: commit messages should be wrapped to 72 columns (see here: https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#44-model-git-commit-messages)

Reply:
Thanks, addressed this